R/snapshot-auto.R

```diff
@@ -1,6 +1,7 @@
 
 the$library_info <- NULL
 
+the$snapshot_failed <- FALSE
```

Comment:
Am I failing to see where this is actually used?

Reply:
You aren't 🙈  I'll fix this.